addons/website_blog/views/snippets/s_latest_posts.xml

```diff
@@ -69,61 +56,70 @@
         </a>
     </figure>
 </template>
-
-<template id="s_latest_posts_big_orizontal_template">
-    <t t-foreach="posts" t-as="p">
-        <figure class="post s_latest_posts_post">
-            <figcaption>
-                <h4 class="mb0"><a t-att-href="'/blog/%s/post/%s' % (p.blog_id.id, p.id)"><t t-esc="p.name"/></a></h4>
-                <h5 class="mt0 mb4" t-field="p.write_date" t-field-options='{"format": "dd/MM"}' />
-            </figcaption>
-            <a t-att-href="'/blog/%s/post/%s' % (p.blog_id.id, p.id)">
+<!-- Horizontal layout -->
+<template id="s_latest_posts_horizontal_template">
```

Comment:
@sdegueldre In fact I think there is a problem since the xml_id changed here, won't this break old snippet compatibility? Worst case, should we make a mapping old xml id -> new xml id in the 000.js file ?

Reply:
Snippets that load templates from the DB are a weird case for our versioning system. Because the new CSS expects the new DOM, but here the snippet can load the new DOM with an old data-vcss version, and apply the wrong css. Maybe in these cases we should version the sub-templates themselves separately from the loader snippet?

Reply:
Not sure. I think it is a matter of case by case here. Ideally, vcss=000 snippets working with the new template work fine so we can drop the old dynamic template. At worst, they don't work with the new template and then we should just keep the old one as compatibility without chaning the xml_id and just finding new xml ids? Which case is it here ?

Worth asking the rest of the team anyway. @JKE-be ? @rdeodoo ?

Reply:
In this case in particular, the original XML hasn't been altered too much because the task was mostly about bringing theme-specific versions into standard and making the different layouts into options, they now just all use a common wrapper. As far as the old css being compatible with the new templates, there is a minor layout problem with one of them (the horizontal version)

Reply:
Ok, then see if you can alter the original css a little to work with the new horizontal version and if not possible I'd say we don't care. The horizontal version was specific to two themes only and it's not a snippet which can be customized by the user anyway (no custom text etc), so if that minor layout problem is a problem to an user, he can simply drop the snippet again.

Reply:
I'm afraid I can't get the old css to work with both versions of the xml. I also think that re-dropping the snippet in this case is not too bad, and not worth duplicating the entire xml with a separate xml_id.